website/docs/integrations/monday.mdx

```diff
@@ -4,39 +4,69 @@ title: monday.com - Manage feature flags from monday
 description: ConfigCat monday app. This is a step-by-step guide on how to connect and manage feature flags from monday.com boards and items.
 ---
 
-Turn features On / Off right from a linked item on your monday.com board with the <a href="https://monday.com/marketplace/10000079" target="_blank">ConfigCat Feature Flags</a> monday.com app. Also you can add targeting or percentage rules just as easily.
+The <a href="https://monday.com/marketplace/10000079" target="_blank">ConfigCat Feature Flags monday app</a> allows you to connect your Monday items and feature flags. Create or link existing flags to your items without leaving your monday instance.
 
-## Installation
+Turn features On/Off right from a linked item on your Monday board. You can also easily modify the linked flags to edit or add new Targeting or Percentage Rules.
```

Comment:
Ez is egy értékajánlat, itt is azt javaslom, hogy ne az Installation szekcióban legyen, hanem a doksumentum elején az első egy-két mondatban.

Reply:
A doksi elején van. Pont az installation szekció került hátrább.